src/ds5/ds5-motion.cpp

```diff
@@ -79,7 +79,7 @@ namespace librealsense
                     assign_stream(_owner->_gpio_streams[p->get_stream_index()-1], p);
                 if (p->get_framerate() == 1000 &&
                     p->get_format() == RS2_FORMAT_MOTION_XYZ32F)
-                    p->make_default();
+                    get_device().tag_profile(p.get());
```

Comment:
Is there a way to ensure that library writer will not forget to call `tag_profile` on one of the camera profiles?

Reply:
actually this "if" should be removed and added is a default stream.
for both this comment and the one above I can init the profiles on the device creation, but I guess that I will need to record everything again.